### DISCUSSION TITLE
simplify Nx workspace.json config

### DISCUSSION THREAD

**Discussion by dongyuwei** [nrwl/nx#7530] [Feature Requests]
Current Nx workspace.json is too long and tedious, can we simplify it like [pnpm's workspace](https://pnpm.io/workspaces) config? Look at Vue3's pnpm workspace  config https://github.com/vuejs/vue-next/blob/master/pnpm-workspace.yaml. 

Also, maybe we can rename Nx workspace.json to nx-workspace.json.

**Comment by miluoshi** [comment]
You can split your workspace.json to per-project configuration to create multiple shorter config files, see: https://blog.nrwl.io/core-performance-improvements-webpack-5-react-native-and-more-in-nx-13-cfe63c7415f1#909e

**Reply by dongyuwei** [reply]
Yes we can use project.json per project, but Nx 13 still supports workspace.json.

**Reply by miluoshi** [reply]
I don't understand, how it does not solve your problem? You are not forced to use single large workspace.json file.

**Reply by dongyuwei** [reply]
can you look at this one ?https://github.com/nrwl/nx/blob/master/workspace.json

**Reply by dongyuwei** [reply]
And compare with https://github.com/vuejs/vue-next/blob/master/pnpm-workspace.yaml
